src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -732,7 +732,6 @@
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\Timer.cs" />
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\Volatile.cs" />
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\WaitHandle.cs" />
-    <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\WaitHandleExtensions.cs" />
```

Comment:
Guess I misread this the first time, I added the check for FeatureCoreClr and resurrected the .cs file.